src/odemis/acq/milling/test/openfibsem_millmng_test.py

```diff
@@ -0,0 +1,85 @@
+# -*- coding: utf-8 -*-
+"""
+Created on Feb 2025
+
+Copyright © Delmic
+
+This file is part of Odemis.
+
+Odemis is free software: you can redistribute it and/or modify it under the terms
+of the GNU General Public License version 2 as published by the Free Software
+Foundation.
+
+Odemis is distributed in the hope that it will be useful, but WITHOUT ANY WARRANTY;
+without even the implied warranty of MERCHANTABILITY or FITNESS FOR A PARTICULAR
+PURPOSE. See the GNU General Public License for more details.
+
+You should have received a copy of the GNU General Public License along with
+Odemis. If not, see http://www.gnu.org/licenses/.
+"""
+import glob
+import logging
+import os
+import unittest
+import time
+import odemis
+from odemis import model
+from odemis.acq.milling.tasks import load_milling_tasks, __file__ as MILLING_PATH
+from odemis.acq.milling.openfibsem import run_milling_tasks_openfibsem, OpenFIBSEMMillingTaskManager
```

Comment:
Can you do something similar as we discussed over chat for tdct, skip the test cases if openfibsem is not copied locally?